### DISCUSSION TITLE
Highlight your bar in the bargraph

### DISCUSSION THREAD

**Discussion by SashaCanfield03** [ppy/osu#29080] [Ideas] [answered]
It isn't hard to see where you among the leaderboard by reading what your score is, but it would be so much more convenient and intuitive if your bar in the bar graph was highlighted to make it stand out. I think gold would be a good color for this, or maybe a different shade of pink.

**Comment by Stoppedpuma** [comment, marked as answer]
Addressed in latest release.
![image](https://github.com/user-attachments/assets/f5b4e3ec-68e6-4b95-a094-3095ad4620c9)
